book/src/slasher.md

```diff
@@ -44,6 +44,7 @@ e.g. `~/.lighthouse/{network}/beacon/slasher_db`. You can use this flag to chang
 directory.
 
 ### Database Backend
+[comment]: <> (lmdb is an option, but lighthouse bn --help shows possible values are mdbx and disabled only, no lmdb. I found that this can be updated in beacon_node/src/cli.rs https://github.com/sigp/lighthouse/blob/693886b94176faa4cb450f024696cb69cda2fe58/beacon_node/src/cli.rs#LL762C1-L768C37, but I don't know how exactly? probably by changing line 767 .possible_values?)
```

Comment:
```suggestion
```

You only get one of these options, depending on which [feature-flags](https://lighthouse-book.sigmaprime.io/installation-source.html?highlight=mdbx#feature-flags) you compile with (`slasher-mdbx` or `slasher-lmdb`).